bson/src/test/unit/org/bson/json/JsonReaderTest.java

```diff
@@ -119,6 +132,16 @@ public void testDateTimeShell() {
     }
 
     @Test
+    public void testDateTimeShellNow() {
+        long currentTime = new Date().getTime();
+        String json = "ISODate()";
```

Comment:
I didn't know the shell allowed calling ISODate without an argument, but I verified that it does.

We don't support ISODate without an argument in .NET, but since the shell allows it we should also.

Note that while Date() returns a string, ISODate() returns a BsonDateTime.